src/libmongoc/src/mongoc/mongoc-cluster-private.h

```diff
@@ -186,6 +186,18 @@ mongoc_cluster_stream_valid (mongoc_cluster_t *cluster, mongoc_server_stream_t *
 bool
 mongoc_cluster_run_command_monitored (mongoc_cluster_t *cluster, mongoc_cmd_t *cmd, bson_t *reply, bson_error_t *error);
 
+// `mongoc_cluster_run_retryable_write` executes a write command and may apply retryable writes behavior.
+// `cmd->server_stream` is expected to be non-owning and is overwritten on retry.
```

Comment:
```suggestion
// `cmd->server_stream` is set to `*retry_server_stream` on retry. Otherwise, it is unmodified.
```
Recommend documenting ownership in the function instead (it's really `mongoc_cmd_t`'s responsibility to document ownership, but alas):
```c
cmd->server_stream = *retry_server_stream; // Non-owning.
```
Non-ownership of `cmd->server_stream` is already implied by documentation of `*retry_server_stream` being required to be cleaned-up on retry.